examples/ChuaCircuit/sim_and_plot.py

```diff
@@ -44,10 +60,40 @@
     MODELS, RESULTS_DIR = write_script(EXPERIMENTS, working_dir=WORKING_DIR,
                                        packages=PACKAGES, fname=FNAME)
 
-    # Ask Dymola to run the script.
-    os.system('dymola ' + FNAME) # For Linux
-    # TODO: Add support for Windows.
-    # os.system(r'C:\Program files\Dymola\bin\Dymola.exe ' + FNAME) # For Windows
+    dymola = None
+
+    def get_last_log(dymola):
+        log = dymola.getLastErrorLog()
+        print(log)
+
+    try:
+        # Instantiate the Dymola interface and start Dymola
+        dymola = DymolaInterface()
+
+
+        # dymola.ExecuteCommand("Advanced.CompileWith64=2")
+        result = dymola.ExecuteCommand('Modelica.Utilities.System.setWorkDirectory("' + Path(WORKING_DIR).as_posix() + '")')
+
+        print('DEBUG1')
+        result = dymola.ExecuteCommand('RunScript("'+Path(os.path.join(os.getcwd(),FNAME)).as_posix()+'", true)')
+
+        print('result code = '+str(result))
+
+        if not result:
+             print("Simulation failed. Below is the translation log.")
+             get_last_log(dymola)
+
+        # dymola.plot(["J1.w", "J2.w", "J3.w", "J4.w"])
+        #dymola.plot(['STABLE_MOTION_G2_PRT.frame_a.f[1]','STABLE_MOTION_G2_PRT.frame_a.f[2]', 'STABLE_MOTION_G2_PRT.frame_a.f[3]'])
+        #dymola.ExportPlotAsImage(os.path.join(dir_result, 'tmp.png'))
```

Comment:
Does not work so far? If so, delete. If it works, add an options at the start, e.g. `PLOT_RESULTS = True`.